app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
@GregP you are refactoring existing code, not changing its behavior (at least not in this PR).  So, you need to continue calling `destroy` and dealing with all the error paths.